<dec f='include/python3.5m/pyerrors.h' l='231' type='PyObject * PyErr_SetFromErrno(PyObject * )'/>
